src/mon/MDSMonitor.cc

```diff
@@ -1954,8 +1972,13 @@ bool MDSMonitor::maybe_expand_cluster(std::shared_ptr<Filesystem> fs)
       break;
     }
 
-    dout(1) << "adding standby " << pending_fsmap.get_info_gid(newgid).addr
+    const auto &new_info = pending_fsmap.get_info_gid(newgid);
+    dout(1) << "adding standby " << new_info.addr
```

Comment:
Since this was just a local log message, I've changed it to match the cluster log one and say "assigned"